MSAL/test/app/ios/MSALTestAppAcquireTokenViewController.m

```diff
@@ -156,11 +156,16 @@ - (void)viewWillAppear:(BOOL)animated
 #pragma mark - Helper
 
 - (MSALPublicClientApplication *)msalTestPublicClientApplication
+{
+    return [self msalTestPublicClientApplicationWithSSOSeeding:NO];
+}
+
+- (MSALPublicClientApplication *)msalTestPublicClientApplicationWithSSOSeeding:(BOOL)ssoSeedingCall
 {
     MSALTestAppSettings *settings = [MSALTestAppSettings settings];
     NSDictionary *currentProfile = [MSALTestAppSettings currentProfile];
-    NSString *clientId = [currentProfile objectForKey:MSAL_APP_CLIENT_ID];
-    NSString *redirectUri = [currentProfile objectForKey:MSAL_APP_REDIRECT_URI];
+    NSString *clientId = ssoSeedingCall ?  @"29d9ed98-a469-4536-ade2-f981bc1d605e" : [currentProfile objectForKey:MSAL_APP_CLIENT_ID];
```

Comment:
can we read this from configuration profile to not include broker clientId and redirect in public source code?

Reply:
Sure I can update this. I was thinking in the real scenario CP can have different clientid/redirectURI then for ssoSeeding call they will use these fixed clientid/redirectURI value so these code reflects that case in our test app. But I forgot about the leaking broker clientid, will update it in the next commit